packages/angular/cli/src/utilities/environment-options.ts

```diff
@@ -14,9 +14,23 @@ function isPresent(variable: string | undefined): variable is string {
   return typeof variable === 'string' && variable !== '';
 }
 
+function isEnabled(variable: string): boolean {
+  return variable === '1' || variable.toLowerCase() === 'true';
+}
+
 const analyticsVariable = process.env['NG_CLI_ANALYTICS'];
 export const analyticsDisabled = isPresent(analyticsVariable) && isDisabled(analyticsVariable);
 
 const analyticsShareVariable = process.env['NG_CLI_ANALYTICS_SHARE'];
 export const analyticsShareDisabled =
   isPresent(analyticsShareVariable) && isDisabled(analyticsShareVariable);
+
+const ciVariable = process.env['CI'];
+export const isCI = isPresent(ciVariable) && isEnabled(ciVariable);
+
+const disableVersionCheckVariable = process.env['NG_DISABLE_VERSION_CHECK'];
+export const disableVersionCheck =
+  isPresent(disableVersionCheckVariable) && isEnabled(disableVersionCheckVariable);
+
+const ngDebugEnvVariable = process.env['NG_DEBUG'];
+export const ngDebug = isPresent(ngDebugEnvVariable) && isEnabled(ngDebugEnvVariable);
```

Comment:
We seem to be doing `isPresent() && isEnabled()` a lot, would it be worth some kind of `envToBoolean('NG_DEBUG')` that just converts the input variable to a regular boolean? That sounds like the utility we actually want here.

Reply:
Yeah that should like a good idea.

Reply:
Yeah good point, encoding the default value somehow would be useful. A couple other ways of doing that could be either `envToBoolean('NG_DEBUG', false /* default */)` or making `envToBoolean()` return `undefined` if not present and then using `??` like `envToBoolean('NG_DEBUG') ?? false`. Up to you exactly how you want to handle it.

Reply:
I updated the usage to remove the `isPresent`, PTAL. I find it clearer to use `isEnabled` and `isDisabled`.